python/deptry/module.py

```diff
@@ -118,10 +119,18 @@ def _get_package_name_from_metadata(self) -> str | None:
         try:
             name: str = metadata(self.name)["Name"]
         except PackageNotFoundError:
+            name = self.is_package_installed(self.name)
+            if name:
+                return name
             return None
         else:
             return name
 
+    def is_package_installed(self, package_name: str):
+        if importlib.util.find_spec(package_name):
+            return package_name
+        return None
```

Comment:
The method can [raise `ModuleNotFoundError` or `ValueError`](https://docs.python.org/3/library/importlib.html#importlib.util.find_spec), so it would be nice to handle those:
```suggestion
    def _is_package_installed(self) -> bool:
        try:
            return find_spec(self.name) is not None
        except (ModuleNotFoundError, ValueError):
            return False
```